Microsoft.ML.AutoML.sln

```diff
@@ -18,6 +18,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tools-local", "tools-local"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.CodeGenerator", "src\Microsoft.ML.CodeGen\Microsoft.ML.CodeGenerator.csproj", "{3148A8CF-A686-4A9C-A5C6-C3E41CAAD64D}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.Recommender", "src\Microsoft.ML.Recommender\Microsoft.ML.Recommender.csproj", "{19D6A195-657E-42D9-A5D4-ECB289BB6FDA}"
+EndProject
```

Comment:
Why `Recommender` appears in AutoML solution?

Reply:
Refer to https://github.com/dotnet/machinelearning/pull/4246/files#diff-466b4b1a71c2aede1d30fa02d412ddc3